src/sections/blog-post.liquid

```diff
@@ -0,0 +1,184 @@
+<div class="page-width">
+  {% if section.settings.title != blank %}
+    <header class="section-header text-center">
+      <h2>{{ section.settings.title | escape }}</h2>
+    </header>
+  {% endif %}
+
+  {%- assign blog = blogs[section.settings.blog] -%}
+
```

Comment:
Maybe a comment here about how the following Liquid is for when we actually have blog items?

Reply:
Added.